.clang-tidy

```diff
@@ -43,9 +43,10 @@ Checks:
   -modernize-use-emplace, -modernize-use-nodiscard, performance-*,
   -performance-unnecessary-value-param, readability-*,
   -readability-convert-member-functions-to-static,
-  -readability-function-cognitive-complexity, -readability-else-after-return,
-  -readability-identifier-length, -readability-implicit-bool-conversion,
-  -readability-magic-numbers, -readability-make-member-function-const,
+  -readability-else-after-return, -readability-enum-initial-value,
```

Comment:
I've been trying to document why various warnings are disabled so that we don't forget. Can you add a note above, something like:

```
  # - readability-enum-initial-value has false positives for `Last = ...`.
```

(it can't be hear due to yaml parsing)

Reply:
Oh yes, thanks. I didn't notice the comments up there.